llvm/lib/Target/DirectX/DXILOpLowering.cpp

```diff
@@ -657,6 +657,14 @@ class OpLowerer {
     return false;
   }
 
+  [[nodiscard]] bool lowerLoadWithStatus(Function &F) {
+    // These should have already been handled in DXILResourceAccess, so we can
+    // just clean up the dead prototype.
+    assert(F.user_empty() && "getpointer operations should have been removed");
+    F.eraseFromParent();
+    return false;
+  }
```

Comment:
This doesn't make sense - this isn't a getpointer operation and it wouldn't be removed. Probably a vestige from an earlier version when you were creating a separate intrinsic that can be removed at this point.